### DISCUSSION TITLE
Is there any way to pass SAS tokens to url parameters?

### DISCUSSION THREAD

**Discussion by calebrob6** [developmentseed/titiler#557] [Q&A] [answered]
If I have titiler running on myserver.com, can I pass COGs stored on Azure blob containers with their SAS tokens as url parameters somehow?

E.g. https://myserver.com/cog/info?url="url_of_a_cog?SAS_token_string"

**Comment by vincentsarago** [comment, marked as answer]
👋 @calebrob6 
https://developmentseed.org/titiler/examples/code/working_with_signed_urls/ this is maybe a bit old but It might help you

**Reply by calebrob6** [reply]
Thanks @vincentsarago!
